CMakeLists.txt

```diff
@@ -1,20 +1,20 @@
-cmake_minimum_required(VERSION 3.16...3.27)
-
-project(rerun_external_cpp_proj LANGUAGES CXX)
-
-set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
-
-if(NOT DEFINED CMAKE_CXX_STANDARD)
-  set(CMAKE_CXX_STANDARD 17)
-endif()
-
-include(FetchContent)
-FetchContent_Declare(rerun_sdk URL https://build.rerun.io/commit/bc05d11/rerun_cpp_sdk.zip) # 2023-10-26
-FetchContent_MakeAvailable(rerun_sdk)
-
-find_package(Eigen3 REQUIRED)
-find_package(OpenCV REQUIRED)
-
-add_executable(rerun_ext_example src/main.cpp)
-
-target_link_libraries(rerun_ext_example Eigen3::Eigen ${OpenCV_LIBS} rerun_sdk)
+cmake_minimum_required(VERSION 3.16...3.27)
+
+project(rerun_external_cpp_proj LANGUAGES CXX)
+
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
+
+if(NOT DEFINED CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 17)
+endif()
+
+include(FetchContent)
+FetchContent_Declare(rerun_sdk URL https://build.rerun.io/commit/b7f404a/rerun_cpp_sdk.zip) # 2023-10-29
+FetchContent_MakeAvailable(rerun_sdk)
+
+find_package(Eigen3 REQUIRED)
+find_package(OpenCV REQUIRED)
+
+add_executable(rerun_ext_example src/main.cpp)
+
+target_link_libraries(rerun_ext_example Eigen3::Eigen ${OpenCV_LIBS} rerun_sdk)

```

Comment:
Added a bunch of `\r` did we? :)

Reply:
urgh.

Reply:
must have been the infamous git autocrlf setting (who thought this is a good idea. why is it not disabled by default)